animation_nodes/nodes/generic/c_utils.pyx

```diff
@@ -0,0 +1,76 @@
+from ... math cimport lerp, mixVec3, mixEul3, mixMat4, mixQuat, mixColor
+from ... data_structures cimport (
+    ColorList,
+    EulerList,
+    DoubleList,
+    Vector3DList,
+    Matrix4x4List,
+    QuaternionList,
+    VirtualColorList,
+    VirtualEulerList,
+    VirtualDoubleList,
+    VirtualVector3DList,
+    VirtualMatrix4x4List,
+    VirtualQuaternionList
```

Comment:
Add a comma after this entry. We do that to make it easier to add entries in the future. The same for similar code.